uni/progs/rankvote.icn

```diff
@@ -0,0 +1,163 @@
+#<p>
+# Implement ranked choice voting.
+#</p>
+#<p>
+# Author: Stephen Wampler (sbw@tapestry.tucson.az.us)
+#</p>
+#<p>
+# This program is in the public domain.
+#</p>
+
+import util
+link printf	     # For sprintf
+
+global ballots,      # Individual ballots
+       totalVotes,   # Total votes for first candidate on each ballot, per round
+       results,      # How each round turned out
+       maxcands      # Most candidates on any ballot (needed in tie-breaks)
+
+#<p>
+#  Read in ranked-choice ballots where each ballot is a single line with the
```

Comment:
(Minor: no need to fix if you don't want to)
Is it worth noting that the program doesn't handle spoilt ballots?